bofire/strategies/doe/objective.py

```diff
@@ -106,16 +106,15 @@ def __init__(
         domain: Domain,
         model: Formula,
```

Comment:
Should we rename `model` to `formula`? This would make the code cleaner and easier to read, or?

Reply:
Done